src/main/java/com/rarchives/ripme/ripper/rippers/ErofusRipper.java

```diff
@@ -59,6 +59,7 @@ public String getGID(URL url) throws MalformedURLException {
     public List<String> getURLsFromPage(Document page) {
         logger.info(page);
         List<String> imageURLs = new ArrayList<>();
+        @SuppressWarnings("unused")
```

Comment:
Don't suppress warnings if possible. Figure out what was intended and update the code accordingly. In this case it might simply be that the variable is unused and was left behind by a past change, and should now be removed. That looks to be the case here.